### ISSUE TITLE
apps not start on my browser 

### ISSUE THREAD

**Issue by polashraj** [taobataoma/meanTorrent#69]
This is my Server Log and apps not start 

2018-06-07T18:30:26.460Z - info:
2018-06-07T18:30:26.460Z - info: Environment:     development
2018-06-07T18:30:26.460Z - info: Server:          http://localhost:3000
2018-06-07T18:30:26.460Z - info: Database:        mongodb://localhost/mean-v2
2018-06-07T18:30:26.460Z - info: App version:     0.1.0
2018-06-07T18:30:26.460Z - info: MEAN.JS version: 0.6.0
2018-06-07T18:30:26.460Z - info: IRC announce:    irc.mean.im:16667 #meanAnnounce
2018-06-07T18:30:26.460Z - info: --
2018-06-07T18:30:28.130Z - info: IRC: connected successfully!
2018-06-07T18:30:28.914Z - info: IRC: join channel #meanAnnounce successfully!

**Comment by taobataoma** [comment]
@polashraj is it a clean install?

**Comment by polashraj** [comment]
no , i have  old mongodb , I Run your script before in this server .

**Comment by taobataoma** [comment]
@polashraj  maybe you should rerun 'npm install' and 'bower install'.

**Comment by polashraj** [comment]
yes i run npm install

**Comment by taobataoma** [comment]
@polashraj Do you change any file? Only you response this issue.

**Comment by polashraj** [comment]
i am edit /config/env/torrents.js

**Comment by taobataoma** [comment]
@polashraj Do you in my skype list? let me to check it.

**Comment by polashraj** [comment]
yes , i am in your skype list , Thank you for your support and beautiful torrent script .

**Comment by taobataoma** [comment]
@polashraj OK，you can talk with me by skype for more private message.
